phoenix-core-client/src/main/java/org/apache/phoenix/compile/CreateIndexCompiler.java

```diff
@@ -113,6 +114,8 @@ private String getValue(PDataType type) {
             return "ARRAY" + type.getSampleValue().toString();
         } else if (type instanceof PJson) {
             return "'{a:1}'";
+        } else if (type instanceof PBson) {
```

Comment:
The previous line for PJson should be corrected. Can we merge two if statements into one with the OR operator?